src/Docker/Runner.php

```diff
@@ -218,7 +218,7 @@ private function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId, Us
         $component = $jobDefinition->getComponent();
         $this->loggersService->setComponentId($component->getId());
 
-        $temp = new Temp("docker");
```

Comment:
Ten `docker` prefix tam byl "jen tak"? Nema to nejaky navaznosti?

Reply:
No byl tam z doby, kdy se každá komponenta (extractor, writer) provozovala jako vlastní služba a měla svůj syrupovej prefix, nebylo to v kontejnerech a všechno se to sypalo na jednu VM, dneska (resp. posledních několik let) už žádný jiný služby nejsou - transformace a orchestrátor mají oddělenej deploy